crates/orderbook/src/main.rs

```diff
@@ -157,6 +160,10 @@ struct Arguments {
         parse(try_from_str = parse_partner_fee_factor),
     )]
     partner_additional_fee_factors: HashMap<AppId, f64>,
+
```

Comment:
Would it make sense to put it into:     shared: shared::arguments::Arguments,?

Reply:
Also, I suggest it being `Option<Url>` since its not necessary to use `Quasimodo` price estimator.

Reply:
Ok, I'll move it.

It makes sense with how we use shared arguments right now, i.e. anything that both solver and orderbook need goes to shared. I happen to have a bit different understanding of shared, i.e. I'd say that shared is anything that any service will need, be that orderbook, solver, or something else that we may write in the future. This is an offtopic discussion though, and I don't think it's a big deal.

Reply:
Oh hm, so in `orderbook` it needs to be optinal, while in `solver` it is mandatory...

Reply:
I think it has the same semantics as the orderbook where it is only mandatory if `Quasimodo` is used as a solver type.